core/dungeon-masters-guide/items/items-wondrous.xml

```diff
@@ -4085,4 +4085,323 @@
+			<p class="indent"><i><b>Mark of Darkness.</b></i> After you spend the requisite amount of time reading and studying the book, you acquire a physical disfigurement as a hideous sign of your devotion to vile darkness. An evil rune might appear on your face, your eyes might become glossy black, or horns might sprout from your forehead. Or you might become wizened and hideous, lose all facial features, gain a forked tongue, or some other feature the DM chooses. The mark of darkness grants you advantage on Charisma (Persuasion) checks made to interact with evil creatures and Charisma (Intimidation) checks made to interact with non-evil creatures.</p>
+			<p class="indent"><i><b>Command Evil.</b></i> While you are attuned to the book and holding it, you can use an action to cast the <em>dominate monster</em> spell on an evil target (save DC 18). You can't use this property again until the next dawn.</p>
+			<p class="indent"><i><b>Dark Lore.</b></i> You can reference the <em>Book of Vile Darkness</em> whenever you make an Intelligence check to recall information about some aspect of evil, such as lore about demons. When you do so, double your proficiency bonus on that check.</p>
+			<p class="indent"><i><b>Dark Speech.</b></i> While you carry the <em>Book of Vile Darkness</em> and are attuned to it, you can use an action to recite word from its pages in a foul language known as Dark Speech. Each time you do so, you take 1d12 psychic damage, and each non-evil creature within 15 feet of you takes 3d6 psychic damage.</p>
+			<p class="indent"><i><b>Destroying the Book.</b></i> The <em>Book of Vile Darkness</em> allows pages to be torn from it, but any evil lore contained on those pages finds its way back into the book eventually, usually when a new author adds pages to the tome.</p>
+			<p class="indent">If a solar tears the book in two, the book is destroyed for 1d100 years, after which it reforms in some dark corner of the multiverse.</p>
+			<p class="indent">A creature attuned to the book for one hundred years can unearth a phrase hidden in the original text that, when translated to Celestial and spoken aloud, destroys both the speaker and the book in a blinding flash of radiance. However, as long as evil exists in the multiverse, the book reforms 1d10 × 100 years later.</p>
+			<p class="indent">If all evil in the multiverse is wiped out, the book turns to dust and is forever destroyed.</p>
+			<div class="reference">
+				<div element="ID_WOTC_DMG_ARTIFACT_TABLE" />
+			</div>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="5">5 lb.</set>
+			<set name="type">Wondrous Items</set>
+			<set name="rarity">Artifact</set>
+			<set name="attunement">true</set>
+		</setters>
+	</element>
+
+	<element name="Eye of Vecna" type="Magic Item" source="Dungeon Master’s Guide" id="ID_WOTC_DMG_MAGIC_ITEM_EYE_OF_VECNA">
```

Comment:
Missing rules for truesight (not sure of the range...)
Add a reference to the ring of X-Ray vision